source/client-side-encryption/corpus/corpus-encrypted.json

```diff
@@ -414,20 +400,6 @@
       }
     }
   },
-  "aws_binData=00_det_auto_altname": {
```

Comment:
"auto" encrypted with "altname" is removed from the schema and data, since we cannot configure a schema with such a field. But "explicit" encrypted with "altname" still exists.